README.md

```diff
@@ -1,26 +1,26 @@
-# fabric-meta
+# legacy-meta
 
-Fabric Meta ia a json HTTP api that can be used to query meta data about fabrics projects. It is updated every 5 mins.
+Legacy Meta is a JSON HTTP API that can be used to query meta data about Legacy-Fabric's projects. It is updated every 5 mins.
 
-I can be used by tools or launchers that wish to query version infomation about fabric.
+It can be used by tools or launchers that wish to query version information about Fabric.
```

Comment:
```suggestion
It can be used by tools or launchers that wish to query version information about Legacy-Fabric.
```

Reply:
What's better, `Legacy Fabric` or `Legacy-Fabric`?

Reply:
In line 3 Deadly used `Legacy-Fabric`, so I used that

Reply:
Ah